zsh/configs/post/path.zsh

```diff
@@ -1,8 +1,10 @@
 # ensure dotfiles bin directory is loaded first
 PATH="$HOME/.bin:/usr/local/sbin:$PATH"
 
-# load rbenv if available
-if command -v rbenv >/dev/null; then
+# load ASDF, falling back to rbenv if not available
+if [ -d "$HOME/.asdf" ]; then
+  . $HOME/.asdf/asdf.sh
```

Comment:
`asdf`s README also suggests:

```
. $HOME/.asdf/completions/asdf.bash
```

Worth it?

https://github.com/asdf-vm/asdf/issues/97#issuecomment-271482373